<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWidget</class>
 <widget class="QWidget" name="QWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_select">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>691</width>
     <height>101</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>에스코어 드림 4 Regular</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>open image</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_image">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>711</width>
     <height>331</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>651</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../datasets/cat_dog/train/cat.166.jpg</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_predict">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>480</y>
     <width>551</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>에스코어 드림 9 Black</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cat</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
